src/epp_topics/python_installation_execution/environment.yml

```diff
@@ -2,6 +2,7 @@
 name: epp
 channels: [conda-forge, nodefaults]
 dependencies:
+  - optimagic
   - estimagic
```

Comment:
Do we need both? Same below.

Reply:
If we want the estimation functions (`estimate_ml` and the like) we need estimagic. Currently you can install estimagic and use it like optimagic, but not the other way around.

Reply:
If that is the case, the first sentence [here](https://optimagic.readthedocs.io/en/latest/estimagic/index.html) is seriously misleading: "estimagic is a subpackage of optimagic ..."

Or are we thinking about a split already in the end? @janosg, any new plans on that? Ideally, I'd make the materials to be prepared today/tomorrow somewhat future-proof...

Reply:
Forget what I said... we will talk